backend/tests/unit/models/test_token_aggregations.py

```
@@ -0,0 +1,392 @@
+"""
+Unit tests for token usage aggregation functionality added in EP-0009.
+
+Tests the computed fields in DetailedStage and SessionStats models that
+aggregate token usage from LLM interactions.
+"""
+
+import pytest
+from unittest.mock import Mock
```

Ruff (0.12.2)

`unittest.mock.Mock` imported but unused

Remove unused import: `unittest.mock.Mock`

(F401)